packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -24,6 +24,10 @@ interface ModalProps {
   onClose: () => void
   show?: boolean
   title?: string
+  /*
+   * Hide the X button if we don't want the user to be able to exit the modal without another action closing the modal
+   */
+  hideCloseXButton?: boolean
```

Comment:
`hideCloseButton` is probably sufficient. If we need to get more granular in the future we can